docs/pages/desktop-access/troubleshooting.mdx

```diff
@@ -32,6 +32,17 @@ If the "Smart Card" service is not running, open PowerShell and run
 `gpupdate.exe /force`. This forces a Group Policy sync and should pick up the
 service changes.
 
+### Smart card not supported for Account
+
+You connect to a Windows host and get the error message: **"Signing in with a smart card isn't supported for your account."**
+or similar. 
+
+**Solution:** Review the Security-Kerberos Log on the Windows host for causes.
+
+The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts.  This is available
+at `Event Viewer->Application and Services Logs->Microsoft->Windows->Security-Kerberos->Operational` in the Windows Event Log.  By default this
+log is not enabled. Enable and attempt to connect with Teleport UI to review log entries.
```

Comment:
```suggestion
The Security-Kerberos Windows Event Log will provide information on Smart Card based authentication attempts. This is available
at `Event Viewer->Application and Services Logs->Microsoft->Windows->Security-Kerberos->Operational` in the Windows Event Log. By default this
log is not enabled. Enable and attempt to connect with Teleport UI to review log entries.
```

Reply:
Does anyone else find the flow a bit awkward here?

It's not enabled by default, but the order in which we describe the solution is 1) check the logs, and 2) enable the logs. Shouldn't it be the other way around?

Reply:
I think it depends on whether or not you need to go down to that log level in order to enable them first or not. I guess I should try it for myself.

Reply:
One does in fact need to traverse down the chain in order to enable the log.

Reply:
@zmb3 ok with that?